watertap/unit_models/tests/test_reverse_osmosis_1D.py

```diff
@@ -308,34 +307,22 @@ def test_build(self, RO_frame):
             "eq_permeate_outlet_isothermal": Constraint,
             "eq_permeate_outlet_isobaric": Constraint,
             "eq_rejection_phase_comp": Constraint,
-            "eq_Kf": Constraint,
-            "eq_N_Re": Constraint,
-            "eq_N_Sc": Constraint,
-            "eq_N_Sh": Constraint,
-            "eq_area_cross": Constraint,
-            "eq_dh": Constraint,
-            "eq_pressure_drop": Constraint,
-            "eq_velocity": Constraint,
-            "eq_friction_factor_darcy": Constraint,
-            "eq_dP_dx": Constraint,
-            "N_Re_avg": Expression,
-            "Kf_avg": Expression,
             "flux_mass_phase_comp_avg": Expression,
         }
-        for (obj_str, obj_type) in unit_objs_type_dict.items():
-            obj = getattr(m.fs.unit, obj_str)
-            assert isinstance(obj, obj_type)
+        # for (obj_str, obj_type) in unit_objs_type_dict.items():
+        #    obj = getattr(m.fs.unit, obj_str)
+        #    assert isinstance(obj, obj_type)
```

Comment:
Yeah we can probably just get rid of these types of tests. If an expected object doesn't exist or isn't the right type, eventually we would get a pyomo error in the build.

Reply:
I agree.